### PR TITLE
Implement commands for listing per-cluster CT/SNAT maps

### PR DESCRIPTION
**Pull request by YutaroHayakawa**
Please see individual commits for more details.

```release-note
Implement commands for listing per-cluster CT/SNAT maps
```

### CODE REVIEW

File: cilium/cmd/bpf_ct_list.go [hunk 1]
```diff
@@ -23,18 +23,23 @@ import (
 // bpfCtListCmd represents the bpf_ct_list command
 var (
 	bpfCtListCmd = &cobra.Command{
-		Use:     "list ( <endpoint identifier> | global )",
+		Use:     "list ( global | endpoint ) [identifier]",
```
Comment (nathanjsweet) [inline cilium/cmd/bpf_ct_list.go]: Why isn't "cluster" documented here?

File: cilium/cmd/bpf_ct_list.go [hunk 2]
```diff
@@ -51,12 +56,46 @@ func init() {
 	command.AddOutputOption(bpfCtListCmd)
 }
 
-func getMaps(eID string) []*ctmap.Map {
-	if eID == "global" {
+func parseArgs(args []string) (string, uint32, error) {
+	t := args[0]
```
Comment (nathanjsweet) [inline cilium/cmd/bpf_ct_list.go:64]: We should check for at least two args to avoid index errors.

**Review by nathanjsweet — CHANGES_REQUESTED** [review]
nit and validation concern.

File: cilium/cmd/bpf_ct_list.go (hunk 1 above)
Comment (YutaroHayakawa) [reply cilium/cmd/bpf_ct_list.go]: Oops, let me fix this.

File: cilium/cmd/bpf_ct_list.go (hunk 2 above)
Comment (YutaroHayakawa) [reply cilium/cmd/bpf_ct_list.go:64]: Good point. I'll fix.

**Comment by YutaroHayakawa** [comment]
Reflected on Nate's comment. And sorry, I added a commit that implements the missing flag string for `from_tunnel` (c.f. https://github.com/cilium/cilium/pull/24212#discussion_r1158217756).

**Comment by YutaroHayakawa** [comment]
Smoke test: https://github.com/cilium/cilium/issues/23812

**Review by nathanjsweet — APPROVED** [review]

**Comment by YutaroHayakawa** [comment]
Cilium Datapath is broken by cilium-cli change (https://github.com/cilium/cilium-cli/pull/1498) and now fixed by https://github.com/cilium/cilium/commit/0a984a71975d63228abfe23fd546f7798b834ef1. I think we don't have to re-run datapath test as it is not related to this change.

**Comment by YutaroHayakawa** [comment]
Rebasing on the master and re-running the CI.

**Comment by YutaroHayakawa** [comment]
ConformanceK8sKind: https://github.com/cilium/cilium/issues/24622

**Pull request merged**